code/world.dm

```diff
@@ -654,8 +654,15 @@ var/f_color_selector_handler/F_Color_Selector
 	bioele_load_stats()
 	bioele_shifts_since_accident++
 	bioele_save_stats()
-
+#ifdef PREFAB_CHECKING
+	placeAllPrefabs()
+#endif
+#ifdef RUNTIME_CHECKING
+	SPAWN_DBG(0)
```

Comment:
Why so soon

Reply:
I just wanted to give `world/proc/init()` a chance to return

Reply:
I would imagine we want to wait 10 seconds or so to see if more runtimes manifest.